src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), SCSS files should be linked as CSS files in HTML. You need to compile your SCSS to CSS and link the compiled CSS file here.